cerberus-dashboard/package.json

```diff
@@ -43,31 +45,24 @@
   },
   "devDependencies": {
     "babel-core": "6.10.4",
-    "babel-eslint": "6.1.0",
-    "babel-loader": "6.2.4",
-    "babel-plugin-syntax-decorators": "6.8.0",
-    "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-preset-es2015": "6.9.0",
-    "babel-preset-react": "6.11.1",
     "babel-preset-stage-1": "6.5.0",
-    "css-loader": "0.23.1",
-    "eslint": "2.13.1",
-    "eslint-html-reporter": "0.5.2",
+    "eslint-html-reporter": "^0.7.4",
     "eslint-loader": "1.3.0",
     "eslint-plugin-react": "5.2.2",
     "estraverse-fb": "1.3.1",
-    "file-loader": "0.9.0",
-    "isparta-loader": "2.0.0",
-    "json-loader": "0.5.4",
-    "lolex": "1.5.0",
-    "node-sass": "3.11.0",
-    "react-addons-test-utils": "15.6.2",
-    "redbird": "0.5.2",
-    "redux-devtools": "3.3.1",
-    "redwire": "2.2.0",
-    "sass-loader": "4.0.0",
-    "style-loader": "0.13.1",
-    "webpack": "1.13.1",
-    "webpack-dev-server": "1.14.1"
+    "react-scripts": "^3.4.0",
+    "redux-devtools": "3.3.1"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
```

Comment:
What's with opera mini?

Reply:
this is the default browser list for `react-scripts`:  https://create-react-app.dev/docs/supported-browsers-features/